lib/parsers/txt.mjs

```diff
@@ -132,6 +161,9 @@ export async function parse (rawText, filename) {
       }
     }
 
+    data.boilerplate.similar2119boilerplate =
+      hasBoilerplateMatch(normalizedText, BOILERPLATE_PARTS.rfc2119, BOILERPLATE_PARTS.rfc2119_alt, BOILERPLATE_PARTS.rfc8174) && (data.boilerplate.rfc2119 || data.boilerplate.rfc8174)
```

Comment:
I still think this should only look for a match of the "rfc2119_alt" and not the other boilerplate blocks. 

(Some history that may help - rfc2119_alt was a variant accepted briefly a _long_ time ago, but it's been acceptable since).